ResidentInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -1,55 +1,133 @@
+using MosaicResidentInformationResponse = ResidentInformationApi.V1.Boundary.Responses.MosaicResidentInformation;
+using PhoneType = ResidentInformationApi.V1.Domain.PhoneType;
+using PhoneTypeResponse = ResidentInformationApi.V1.Boundary.Responses.PhoneType;
 
 namespace ResidentInformationApi.V1.Factories
 {
     public static class ResponseFactory
     {
+        public static AcademyClaimantInformationResponse ToResponse(this AcademyClaimantInformation claimants)
+        {
+            return new AcademyClaimantInformationResponse
+            {
+                ClaimId = claimants.ClaimId,
+                PersonRef = claimants.PersonRef,
+                FirstName = claimants.FirstName,
+                LastName = claimants.LastName,
+                DateOfBirth = claimants.DateOfBirth,
+                NINumber = claimants.NINumber,
+                ClaimantAddress = claimants.ClaimantAddress.ToResponse(),
+                CheckDigit = claimants.CheckDigit
+            };
+        }
+
         public static HousingResidentInformationResponse ToResponse(this HousingResidentInformation domain)
         {
             return new HousingResidentInformationResponse
             {
-                // System = "academy", //TODO
-                // SystemId = "123456", //TODO
-                // SystemUrl = new System.Uri("https://academy-api.hackney.gov.uk"), //TODO
+                HouseReference = domain.HouseReference,
+                PersonNumber = domain.PersonNumber,
+                TenancyReference = domain.TenancyReference,
+                NationalInsuranceNumber = domain.NationalInsuranceNumber,
                 FirstName = domain.FirstName,
                 LastName = domain.LastName,
                 NhsNumber = domain.NhsNumber,
                 DateOfBirth = domain.DateOfBirth,
                 Uprn = domain.Uprn,
                 AddressList = domain.AddressList?.ToResponse(),
-                PhoneNumber = domain.PhoneNumberList?.ToResponse()
+                PhoneNumber = domain.PhoneNumberList?.ToResponse(),
+                EmailAddressList = domain.EmailAddressList?.ToResponse()
             };
         }
-        public static List<HousingResidentInformationResponse> ToResponse(this IEnumerable<HousingResidentInformation> residents)
+
+        public static MosaicResidentInformationResponse ToResponse(this MosaicResidentInformation domain)
         {
-            return residents.Select(r => r.ToResponse()).ToList();
+            return new MosaicResidentInformationResponse
+            {
+                MosaicId = domain.MosaicId,
+                FirstName = domain.FirstName,
+                LastName = domain.LastName,
+                Uprn = domain.Uprn,
+                DateOfBirth = domain.DateOfBirth,
+                PhoneNumber = domain.PhoneNumberList.ToResponse(),
+                AddressList = domain.AddressList.ToResponse(),
```

Comment:
Are PhoneNumberList and AddressList nullable in the Mosaic API response? If so you will need to add questions marks here like in the ToResponse method for housing data above.

Reply:
The nullable items in Housing were there before I started work on this branch. 
According to the SwaggerDocs, there's nothing specifically nullable listed in either Housing or Mosaic, but @louiseswift created specific tests within the response factory to check that Housing could return a record when Address, Phone and Email were nullable. I will check with her why those were added.